src/python_be.cc

```diff
@@ -1718,6 +1719,14 @@ ModelState::ModelState(TRITONBACKEND_Model* triton_model)
       }
     }
 
+    triton::common::TritonJson::Value platform;
+    if (model_config_.Find("platform", &platform)) {
+      auto error = platform.AsString(&platform_);
```

Comment:
Can we add a check to disable platforms other than `tensorflow_savedmodel` for now as discussed?

Reply:
I think we wanted to keep for backwards compat - but we can put the check in the model context for handlers?

Reply:
platform for python backend is a new functionality so I don't think there would be any backward compat issues.

Reply:
@Tabrizian  The field always existed. There might be users using the field value for something specific in their model.py. The only check we have right now is to look for the presence of directory within platform_handlers. If found, then execute the python model. If not, then fallback to previous logic.

Reply:
I wouldn't worry about that too much. This is not part of the API that we offer to the Python backend users. The platform field is for the core to consume not for the Python model developer.

Reply:
For disabling platform field for values other than "tensorflow_savedmodel", I believe it is ok to leave the logic as is. We are only installing tensorlfow_savedmodel in the platform handler. We would fallback to the previous implementation for all the rest field values.